Duplicati/Library/Backend/TencentCOS/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net462" />
+  <package id="Tencent.QCloud.Cos.Sdk" version="5.4.11" targetFramework="net462" />
```

Comment:
We should add the license information, etc. for `Tencent.QCloud.Cos.Sdk` to the `Duplicati.License` project.